ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -143,6 +143,27 @@ static LK_PATCH: Mutex<()> = Mutex::new(());
 /// x86 NOP opcode.
 const NOP_OPCODE: u8 = 0x90;
 
+/// Returns the offset of the given thread local in relation to the segment register `fs`. At JIT
+/// runtime we use this offset to calculate the absolute address of the thread local for each
+/// thread.
+fn get_tls_offset(name: &std::ffi::CString) -> i32 {
```

Comment:
Is there a reason for the signedness here? Presumably it's always a -ve offset?

Reply:
Let's document that and add an `assert` guaranteeing that. [Or change it to a `u32`, but I take your point about dynasm.]

Reply:
[Feel free to squash those changes straight down.]

Reply:
Fixed.